lib/modules/datasource/maven/readme.md

```diff
@@ -0,0 +1,16 @@
+### Fetching changelogs from API
+
+In case your repository doesn't hold a `CHANGELOG.md` file and you need to fetch changelogs from an API (just like the one from [Gitlab](https://docs.gitlab.com/ee/api/releases) or [Github](https://docs.github.com/en/rest/releases)), you need to configure `Maven` so that `renovatebot` is able to determine the `sourceUrl` when fetching info about the artifact.
+
+To do so, you need to configure the [scm section](https://maven.apache.org/scm/git.html) on the `pom.xml` of the given artifact.
```

Comment:
It sounds like the goal with this sentence is to explain that you put the path to your repository in the `pom.xml` of the given artifact?

Reply:
Yes, exactly.